package.json

```diff
@@ -1,20 +1,42 @@
 {
   "name": "hackapp-client",
-  "version": "0.0.1",
-  "description": "",
-  "main": "src/index.js",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.9.1",
+    "@material-ui/icons": "^4.9.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "eslint-plugin-flowtype": "^3.13.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "3.3.1",
+    "typescript": "^3.7.5"
```

Comment:
Interesting dep! Are you going to be developing in typescript?

Reply:
I found that strange too! It's a dependency for some material UI elements—when I was installing material, it requested that I install typescript. I haven't actually needed it yet (as far as I am aware). I'll look into whether or not I can remove this dependency for a future PR!

Reply:
Looks like that was my bad—I misread one of the errors when installing packages. I took out that dependency, should be good now!